Framework/Algorithms/src/FFT.cpp

```diff
@@ -145,10 +154,8 @@ void FFT::exec() {
     }
   }
 
-  // centerShift == true means that the zero on the x axis is assumed to be in
-  // the data centre
-  // at point with index i = ySize/2. If shift == false the zero is at i = 0
-  const bool centerShift = true;
+  // Zero on the x axis is assumed to be in the centre,
+  // at point with index i = ySize/2.
 
```

Comment:
Agreed, I've put `centerShift` back to make the code clearer. The default behaviour (to assume the data is centered at x=0 unless *Shift* or *AutoShift* is used) is a bit confusing but should be kept the same for compatibility. I've tried to make this a bit clearer in the documentation.